app/services/UpdateMongoCacheService.scala

```diff
@@ -76,14 +77,14 @@ class UpdateMongoCacheService @Inject()(http: HttpClient, val cacheConnector: Da
   def getMongoCacheData(fileName: String)(implicit hc: HeaderCarrier, ex: ExecutionContext): Future[Option[UpdateMongoCacheResponse]] = {
     val requestUrl = s"${applicationConfig.mongoCacheUpdateUrl}$fileName"
 
-    http.GET[UpdateMongoCacheResponse](requestUrl)
+    http.GET[UpdateMongoCacheResponse](requestUrl, Seq.empty, Seq.empty)
       .map { r =>
         import utils.Strings._
         println(Json.prettyPrint(Json.toJson(r)) in Console.YELLOW)
         Some(r.copy(dataImport = Some(DataImport(fileName)))) }
       .recover {
-        case e => throw e
         case _: NotFoundException => None
```

Comment:
I wonder if this also needs to be UpstreamError and maybe removing empty headers and query